tests/kernel/common/prj.conf

```diff
@@ -6,4 +6,4 @@ CONFIG_BOOT_DELAY=500
 CONFIG_IRQ_OFFLOAD=y
 CONFIG_TEST_USERSPACE=y
 CONFIG_BOUNDS_CHECK_BYPASS_MITIGATION=y
-CONFIG_MAX_THREAD_BYTES=3
+CONFIG_MAX_THREAD_BYTES=4
```

Comment:
do not change the prj config, instead create a board folder and use board.conf to override it.

Reply:
Yes, will update it. Thanks for the good suggestion!